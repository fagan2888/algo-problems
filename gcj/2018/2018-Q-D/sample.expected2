Case #1: 
0.5000000000 0.0000000000 0.0000000000
0.0000000000 0.5000000000 0.0000000000
0.0000000000 0.0000000000 0.5000000000
Case #2: 
0.4857023073 0.1178509427 0.0142976927
-0.1178509427 0.4714046145 0.1178509427
0.0142976927 -0.1178509427 0.4857023073
Case #3: 
0.4857022604 0.1178511302 0.0142977396
-0.1178511302 0.4714045208 0.1178511302
0.0142977396 -0.1178511302 0.4857022604
