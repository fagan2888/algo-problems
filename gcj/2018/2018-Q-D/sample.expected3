Case #1: 
0.5000000000 0.0000000000 0.0000000000
0.0000000000 0.5000000000 0.0000000000
0.0000000000 0.0000000000 0.5000000000
Case #2: 
0.4850713011 -0.1178509427 0.0285830039
0.1212676083 0.4714046145 -0.1143322203
0.0000000000 0.1178509427 0.4859127034
Case #3: 
0.4850712501 -0.1178511302 0.0285830975
0.1212678125 0.4714045208 -0.1143323901
0.0000000000 0.1178511302 0.4859126579
