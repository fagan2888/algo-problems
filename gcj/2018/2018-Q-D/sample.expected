Case #1: 
0.3535533906 0.0000000000 0.3535533906
0.0000000000 0.5000000000 -0.0000000000
-0.3535533906 0.0000000000 0.3535533906
Case #2: 
0.3535533906 0.1178509427 0.3333333996
0.0000000000 0.4714046145 -0.1666664016
-0.3535533906 0.1178509427 0.3333333996
Case #3: 
0.3535533906 0.1178511302 0.3333333333
0.0000000000 0.4714045208 -0.1666666667
-0.3535533906 0.1178511302 0.3333333333
